### PR TITLE
WIP JSON Payment Protocol V2 - Transmit Signed Tx - Multiple Outputs

### PR DESCRIPTION
**Pull request by stevenzeiler**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/actions/BitPayActions.js [hunk 1]
```diff
@@ -158,12 +158,12 @@ export async function launchBitPay(
   const requiredFeeRate = invoiceInstruction.requiredFeeRate
   const spendInfo: EdgeSpendInfo = {
     selectedCurrencyCode,
-    spendTargets: [
-      {
-        nativeAmount: instructionOutput.amount.toString(),
```
Comment (paullinator) [inline src/actions/BitPayActions.js:163]: Need to remove line 155. `yarn precommit` should have caught this with lint.

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/actions/BitPayActions.js (hunk 1 above)
Comment (stevenzeiler) [reply src/actions/BitPayActions.js:163]: Okay thanks I will use yarn precommit

**Comment by paullinator** [comment]
Looks good. Let me know when fully tested

**Comment by paullinator** [comment]
@thehobbit85  can you help with the build issues

**Pull request closed without merging**